dots/bashrc

```diff
@@ -74,16 +75,16 @@ parse_git_branch() {
   __git_ps1 " [%s]"
 }
 
-parse_ruby_version() {
-  echo " [$(rbenv version-name)]"
+parse_versions() {
+  echo " [rb:$(rbenv version-name)|ex:$(exenv version-name)]"
```

Comment:
Not sure if there's a nicer way to display this - it's not super relevant outside of a project in that language, but detecting is a little bit of a pain